### PR TITLE
TSD-2794 <Heading /> unit testing.

### PR DESCRIPTION
**Pull request by DanMad**

### CODE REVIEW

File: lib/badge/badge.test.tsx [hunk 1]
```diff
@@ -40,6 +40,11 @@ describe('<Badge />', () => {
 
     wrapper.setProps({ style: { zIndex: '1' } });
```
Comment (victorliafook) [inline lib/badge/badge.test.tsx]: s there a point to test zIndex here again, or is it just duplication?

File: lib/divider/divider.test.tsx [hunk 2]
```diff
@@ -48,6 +48,11 @@ describe('<Divider />', () => {
 
     wrapper.setProps({ style: { zIndex: '1' } });
```
Comment (victorliafook) [inline lib/divider/divider.test.tsx]: same here

File: lib/uniform/uniform.test.tsx [hunk 3]
```diff
@@ -40,20 +58,32 @@ describe('<Uniform />', () => {
...
+  });
 
-    expect(wrapper.getDOMNode().getAttribute('class')).toBe(
-      'oc-no-touchevents'
+  it('Handles props.style', () => {
+    wrapper = mount(<Uniform>foo</Uniform>);
+
+    expect(wrapper.getDOMNode().getAttribute('style')).toBe(null);
+
+    wrapper.setProps({ style: { zIndex: '1' } });
```
Comment (victorliafook) [inline lib/uniform/uniform.test.tsx]: same here... ignore if its useful indeed

File: lib/badge/badge.test.tsx (hunk 1 above)
Comment (DanMad) [reply lib/badge/badge.test.tsx]: I'm testing 0, 1 and 2 CSS properties across all the assertions—just to be thorough.

File: lib/divider/divider.test.tsx (hunk 2 above)
Comment (DanMad) [reply lib/divider/divider.test.tsx]: I'm testing 0, 1 and 2 CSS properties across all the assertions—just to be thorough.

File: lib/uniform/uniform.test.tsx (hunk 3 above)
Comment (DanMad) [reply lib/uniform/uniform.test.tsx]: I'm testing 0, 1 and 2 CSS properties across all the assertions—just to be thorough.

**Pull request merged**